### PR TITLE
Add associated Legendre Polynomials P_l,m

### PR DESCRIPTION
**Pull request by eschnett**

### CODE REVIEW

**Comment by jishnub** [comment]
I'll fix it in a separate PR so that this isn't cluttered

File: src/LegendrePolynomials.jl [hunk 1]
```diff
@@ -303,6 +404,72 @@ julia> collectPl(0.5, lmax = 4)
 """
 collectPl(x; lmax::Integer) = collect(LegendrePolynomialIterator(x, lmax))
 
+"""
+	collectPlm(x; n::Integer, lmax::Integer)
```
Comment (jishnub) [inline src/LegendrePolynomials.jl:408]: should this 'n' be 'm' instead?

**Comment by jishnub** [comment]
Thanks for this, it's a useful addition.

File: src/LegendrePolynomials.jl (hunk 1 above)
Comment (eschnett) [reply src/LegendrePolynomials.jl]: It should; thanks.

The order of arguments was also wrong.

**Comment by jishnub** [comment]
Is this good to go? Could you bump the patch version? In that case I'll merge and tag a new version

**Comment by eschnett** [comment]
Yes, this is good to go. I updated the version number.

**Comment by jishnub** [comment]
Thanks again!

**Pull request merged**